test/e2e/latency.go

```diff
@@ -0,0 +1,271 @@
+/*
```

Comment:
@wojtek-t

Yep, I will refactor it -- this is really still a WIP.  The idea is to
maintain the current behavior (test latency during density tests on a
primed cluster) and also be able to run independently of the density tests.

On Tue, Sep 15, 2015 at 3:13 AM, Wojciech Tyczynski <
notifications@github.com> wrote:

> In test/e2e/latency.go
> <https://github.com/kubernetes/kubernetes/pull/13932#discussion_r39480490>
> :
>
> > @@ -0,0 +1,271 @@
> > +/*
>
> Can you please refactor it somehow (maybe by putting the main logic to the
> new "perf_util.go" file) so that we can just measure the startup latency in
> Density test too?
> It's quite important that we don't add a new test to our scalability suite
> now (as long as deleting namespace after them takes 30m +).
>
> Ideally, how I would see it, is that we have some helper function:
> "testPodStartupLatency(client, /*maybe sth else*/)"
> that we could call from both Density tests and possibly from the new test
> you created here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13932/files#r39480490>.
>